src/index.js

```diff
@@ -63,3 +64,7 @@ export default (options: options = {}): Promise<any> => {
   });
 
 };
+
+animateValue.easings = Object.assign({}, namedEasings);
+
```

Comment:
I'm iffy about this. As a user I'd like to access the easing functions directly (to compose my own custom easings if desired). Having it as a separate export in index would break the api (users would have to look for exports.default in cjs), and attaching it to the default function means it can be found in the minified dist script as well (e.g. window.animateValue.easings). But it just also kind of feels ... weird. Thoughts?